processing/src/test/java/org/apache/druid/query/aggregation/first/FloatFirstAggregationTest.java

```diff
@@ -91,7 +95,25 @@ public void testDoubleFirstAggregator()
   }
 
   @Test
-  public void testDoubleFirstBufferAggregator()
+  public void testFloatFirstAggregatorWithTimeColumn()
+  {
+    Aggregator agg = new FloatFirstAggregatorFactory("billy", "nilly", "customTime").factorize(colSelectorFactory);
+
+    aggregate(agg);
```

Comment:
nit: Should this be done in a loop?

Reply:
at 4 calls, a loop would save 1 line.  Who knows which is actually more better.  There is a simplicity to just having the same thing written 4 times.

Reply:
keeping it as is for now